help-center/connector-client.mdx

```diff
@@ -0,0 +1,161 @@
+- [Google Drive](https://www.elastic.co/guide/en/enterprise-search/master/connectors-google-drive.html)
+- [GraphQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-graphql.html)
+- [Jira](https://www.elastic.co/guide/en/enterprise-search/master/connectors-jira.html)
+- [MicrosoftSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-ms-sql.html)
+- [MongoDB](https://www.elastic.co/guide/en/enterprise-search/master/connectors-mongodb.html)
+- [MySQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-mysql.html)
+- [Network drive](https://www.elastic.co/guide/en/enterprise-search/master/connectors-network-drive.html)
+- [Notion](https://www.elastic.co/guide/en/enterprise-search/master/connectors-notion.html)
+- [OneDrive](https://www.elastic.co/guide/en/enterprise-search/master/connectors-onedrive.html)
+- [OpenText Documentum](https://www.elastic.co/guide/en/enterprise-search/master/connectors-opentext.html)
+- [Oracle](https://www.elastic.co/guide/en/enterprise-search/master/connectors-oracle.html)
+- [Outlook](https://www.elastic.co/guide/en/enterprise-search/master/connectors-outlook.html)
+- [PostgreSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-postgresql.html)
+- [Redis](https://www.elastic.co/guide/en/enterprise-search/master/connectors-redis.html)
+- [S3](https://www.elastic.co/guide/en/enterprise-search/master/connectors-s3.html)
+- [Salesforce](https://www.elastic.co/guide/en/enterprise-search/master/connectors-salesforce.html)
+- [ServiceNow](https://www.elastic.co/guide/en/enterprise-search/master/connectors-servicenow.html)
+- [SharePoint Online](https://www.elastic.co/guide/en/enterprise-search/master/connectors-sharepoint-online.html)
+- [SharePoint Server](https://www.elastic.co/guide/en/enterprise-search/master/connectors-sharepoint.html)
+- [Slack](https://www.elastic.co/guide/en/enterprise-search/master/connectors-slack.html)
+- [Teams](https://www.elastic.co/guide/en/enterprise-search/master/connectors-teams.html)
+- [Zoom](https://www.elastic.co/guide/en/enterprise-search/master/connectors-zoom.html)
+</DocAccordion>
+
+<DocCallOut color="warning" title="Data source prerequisites">
+Each data source has specific prerequisites to authorize the connector to access its data.
+For example, certain data sources may need you to create an OAuth application, or create a service account.
+You'll need to check the [individual connector documentation](#available-connectors) for these details.
+</DocCallOut>
+
+## Create an API key
+
+The connector service needs an API key to authenticate with ((es)).
+If you already have one you're good to go.
+If not you can create a key in your ((es)) project's UI.
+
+Search for **API keys** in the top-level search bar and follow the instructions.
+Copy the `encoded` value of the API key, which you'll need in the next steps.
+
+## Initial setup in UI
```

Comment:
This was just because the user needs to move between the UI and terminal but let's just go with "Initial setup" because we won't be deploying anything in the UI